aptos-move/block-executor/src/executor.rs

```diff
@@ -54,12 +56,123 @@ use std::{
     sync::{atomic::AtomicU32, Arc},
 };
 
+pub struct ProcessedAccumulator<T: Transaction> {
```

Comment:
nit; name is mildly confusing, maybe `ProcessedTxnInfoAccumulator`, `RollingCommitInfoAccumulator`, `BlockGasLimitProcessor`